docs/_pages/gallery/gallery_tutorial_modelling_magnets.md

```diff
@@ -6,79 +6,95 @@ orphan: true
 
-To find the correct magnetic polarization of a magnet from the data sheet, one must find the crossing between B-H curve and respective permeance coefficient line. This gives the "working point" which expresses the mean opposing H-field inside magnet. The correct polarization to use in the Magpylib simulation is then simply the J-value at the working point taken from the J-H curve.
+To obtain the correct magnetic polarization of a magnet from the data sheet, one must find the crossing between B-H curve and respective permeance coefficient line. This gives the "working point" which corresponds to the mean demagnetizing H-field inside the magnet. The correct polarization to use in the Magpylib simulation is the J-value at the working point which can be read off from the J-H curve.
 
-![Alt text](../../_static/images/gallery_tutorial_magnet_datasheet2.png)
+![data sheet snippet](../../_static/images/gallery_tutorial_magnet_datasheet2.png)
 
-In the following sections we give background information that explains in more detail why it is like this.
+In the following sections we explain in more detail why it is like this.
 
 ## Hysteresis loop
 
-If you've worked with magnetism, chances are very high that you have seen a magnetic hysteresis loop. The normal loop describes the connection between the **mean values** of an externally applied H-field and the resulting B-field **within a defined volume**. The intrinsic loop describes the connection between the H-field and the magnetic polarization $J$ (or magnetization $M$). This connection depends strongly on size and shape of this volume and what is inside and what is outside. The two loops are connected via $B = \mu_0 H + J$. In free space the B-H "loop" is just a straight line defined via $B = \mu_0 H$. Obviously, the J-H "loop" is just zero everywhere. When the whole space is filled with magnetic material you will see something like this within an arbirary volume,
+If you've worked with magnetism, chances are very high that you have seen a magnetic hysteresis loop. Hysteresis loops describe the connection between the **mean values** of an externally applied H-field and the resulting B-field, polarization J or magnetization M **within a defined volume**. This connection depends strongly on size and shape of this volume and what is inside and what is outside.
+
+The B-H curve is called the "normal loop", while J-H and M-H curves are called "intrinsic loops". Hereon we only make use of the J-H loops, but the discussion is similar for M-H. Normal and intrinsic loops are connected via $B = \mu_0 H + J$. In free space the B-H connection is just a straight line defined via $B = \mu_0 H$. When the whole space is filled with magnetic material you will see something like this within an arbirary volume,
 
 ::::{grid} 2
```

Comment:
@OrtnerMichael 
Please don't use the grid admonition just to center an resize an image. There are much more straightforward syntaxes for this. Also it allows to reference the image in the text and show a preview while hovering over the reference.
Have a look the myst parser docs (https://myst-parser.readthedocs.io/en/latest/syntax/images_and_figures.html) 
;)